cli/azd/resources/apphost/templates/resources.bicept

```diff
@@ -309,10 +309,12 @@ resource {{bicepName $name}}RoleAssignment 'Microsoft.Authorization/roleAssignme
   }
 }
 {{- end}}
+
 {{range $name, $value := .CosmosDbAccounts}}
-resource {{bicepName $name}} 'Microsoft.DocumentDB/databaseAccounts@2023-11-15' = {
+resource {{bicepName $name}} 'Microsoft.DocumentDB/databaseAccounts@2023-04-15' = {
   name: replace('{{$name}}-${resourceToken}', '-', '')
   location: location
+  kind: 'GlobalDocumentDB'
```

Comment:
why are we adding this?

Reply:
the Aspire provisioner ( https://github.com/dotnet/aspire/blob/main/src/Aspire.Hosting.Azure.Provisioning/Provisioners/AzureCosmosDBProvisioner.cs#L66) is not setting this explicitly. I'm guessing that might be the default `kind` .

I would still recommend checking with @mitchdenny about this. Azd must produce the same result as the aspire provisioner.

Reply:
cc @Pilchie

Reply:
My team doesn't own the management SDK, but I expect that it is the default.  This question talks about the values a bit: https://stackoverflow.com/questions/70613680/how-does-azure-cosmos-db-apis-maps-in-cli-and-azure-portal